tests/macsec/test_deployment.py

```diff
@@ -8,7 +9,7 @@
 
 pytestmark = [
     pytest.mark.macsec_required,
-    pytest.mark.topology("t0", "t2"),
+    pytest.mark.topology("t2"),
```

Comment:
Hello @Azarack we cannot remove t0 here as we still run the macsec tests for PR sanity with t0 toplogy

Reply:
I added it back.  Can you check the pipeline as everything is failing on the T0 run with no real error message.  Both this PR and others I have recently submitted.